### PR TITLE
chore: add scoped package

### PR DESCRIPTION
**Pull request by tbozhikov**

### CODE REVIEW

**Comment by tbozhikov** [comment]
test  build_nativescript_dev_webpack#ns-scope-in-host-resolver

**Comment by tbozhikov** [comment]
test build_nativescript_dev_webpack#ns-scope-in-host-resolver

**Comment by tbozhikov** [comment]
test branch_nativescript_dev_webpack#ns-scope-in-host-resolver

**Comment by tbozhikov** [comment]
test branch_nativescript_dev_webpack#ns-scope-in-host-resolver

File: nativescript-angular/router/router.module.ts [hunk 1]
```diff
@@ -15,9 +15,8 @@ import { NSEmptyOutletComponent } from "./ns-empty-outlet.component";
 export { PageRoute } from "./page-router-outlet";
 export { RouterExtensions } from "./router-extensions";
 export { NSModuleFactoryLoader } from "./ns-module-factory-loader";
-export { NSEmptyOutletComponent } from "./ns-empty-outlet.component";
 
-export type LocationState = LocationState;
```
Comment (VladimirAmiorkov) [inline nativescript-angular/router/router.module.ts]: Is this a breaking change, from where do we export this?

File: build/pack-scripts/pack-scoped.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import * as path from "path";
+import * as fs from "fs-extra";
+import { execSync } from "child_process";
+
+console.log(`Packing @nativescript/angular package`);
+
+const distFolderPath = path.resolve("../../dist");
```
[the remark this answers is not in the document]
Comment (tbozhikov) [reply build/pack-scripts/pack-scoped.ts]: We use common dist folder for consistency with @nativescript/core and tns-core-modules repo.

File: nativescript-angular-package/router/router.module.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export { LocationState } from "@nativescript/angular/router/ns-location-strategy";
```
[the remark this answers is not in the document]
Comment (tbozhikov) [reply nativescript-angular-package/router/router.module.ts]: 👍 will fix

File: nativescript-angular/router/router.module.ts (hunk 1 above)
Comment (tbozhikov) [reply nativescript-angular/router/router.module.ts]: it is not, this change will go in the new scoped package only, the compatibility one will have the type exported from this path

**Pull request merged**